core/trino-main/src/main/java/io/trino/sql/routine/SqlRoutinePlanner.java

```diff
@@ -123,6 +123,8 @@ private class StatementVisitor
         private final Analysis analysis;
         private final StandardFunctionResolution resolution;
 
+        private int labelCounter;
```

Comment:
Make this `final AtomicInteger` and then we don't need a method. We do this in other places in the planner.

Reply:
fixed!